DMPRoadmap/roadmap
# [Don't merge yet] Integrate Zenodo API for Plan PDF uploads

Changes proposed in this PR:
- Add support for users to connect their account to [Zenodo API](http://developers.zenodo.org/)
- Add support for users to upload a DMP to Zenodo as a PDF

Please note: to test this, you'll need to do the following:

1. Add a `HOST` env variable (for OAuth 2.0 redirects). This should be set to whatever master domain the app is running on (will be 'localhost:3000' in development).
2. Create an application on the [Zenodo sandbox](https://sandbox.zenodo.org/) and [Zenodo live](https://zenodo.org/) environments 
3. Obtain.a `CLIENT_ID` and `CLIENT_SECRET` for the applications, and store these as env variables.

Go to the "Share" tab for a given Plan and click the Zenodo button at the foot of the page
